internal/log/log.go

```diff
@@ -75,7 +75,7 @@ func CreateLogger(
 		}
 		bootstrapLogger.Info("Log file path provided", zap.String("path", c.LogSink))
 		if c.StdOut || foreground {
-			output = zap.Output(zap.MultiWriteSyncer(os.Stdout, sink))
+			output = zap.Output(zap.Tee(os.Stdout, sink))
```

Comment:
Any specific reason for reverting this back  to the older version? I am afraid the `go get` might fail again.